jupyter/jupyter-web-app/base/kustomization.yaml

```diff
@@ -10,6 +10,7 @@ resources:
 - service-account.yaml
 - service.yaml
 - virtual-service.yaml
+- application.yaml  
```

Comment:
remove trailing black spaces on all kustomization.yamls and then do `make generate; make test` then tests will pass.

This has overlap with #67 but I can resolve when that goes in